tests/acceptance/features/webUISharingInternalUsersToRootPreviews/shareWithUsers.feature

```diff
@@ -49,3 +49,18 @@ Feature: Sharing details and preview of files
     When the user opens the share dialog for file "lorem.txt" using the webUI
     Then user "Brian Murphy" should be listed as "Editor" via "simple-folder" in the collaborators list on the webUI
     And user "Carol King" should be listed as "Editor" via "sub-folder" in the collaborators list on the webUI
+
+
+  Scenario: file list view image preview in file share
+    Given user "Alice" has uploaded file "testavatar.jpg" to "testavatar.jpg"
+    And user "Alice" has shared file "testavatar.jpg" with user "Brian"
+    When user "Brian" logs in using the webUI
+    Then the preview image of file "testavatar.jpg" should be displayed in the file list view on the webUI
```

Comment:
IMO various previews test were disabled (and maybe deleted?) last week. The tests were changed to run with previews disabled, because of flaky tests.

Are you intending to put those back?

Reply:
It seems to be a mistake, I will remove this